RELEASE.md

```diff
@@ -0,0 +1,117 @@
+## Automated Release
+
+Release process:
+
+- Verify the release version by looking at the [releases page](https://github.com/kanisterio/kanister/releases) on the Kanister repo.
+- Trigger the [pre-release](#pre-release-workflow) workflow with the desired version number (e.g. bump the minor version portion 0.106.0 -> 0.107.0)
+- Review and validate created PR that it doesn't have any unintended changes
+- Make sure to validate that all merged PRs in the release have [release notes](#release-notes)
+	- Make sure that CHANGELOG.md and CHANGELOG_CURRENT.md contain release notes for the release version
+	- **NOTE** While we establish the new process of release notes, it may be required to add notes in pre-release step by commiting them into pre-release branch
+- Approve and merge the pre-release PR (it will be merged by `kueue` when approved)
+- Merging of pre-release PR will trigger the `release.yaml` pipeline, which will create a github release and publish the images 
+- The Kanister release job will publish a new tag, update documentation, build all the docker images, and push them to the [ghcr.io](https://github.com/orgs/kanisterio/packages) registry.
+- Once the job is complete, a Slack notification will be sent to the kanister channel.
+- Post release announcement in kanister slack and https://groups.google.com/g/kanisterio 
+
+### Pre-release workflow
+
+`pre-release` workflow serves to create a new version of kanister, it updates version number and creates a PR in kanister repo.
+
+The workflow can be triggered using workflow dispatch from the `Actions` tab in the repo: https://github.com/kanisterio/kanister/actions/workflows/pre-release.yml
+It has a required input of `release_tag`, which should be set to a version next to the current version.
+
+This will result in creating a PR with a version bump, like https://github.com/kanisterio/kanister/pull/2629
+**NOTE** PR description would look like `pre-release: Update version to ...`
+
+This PR would contain an update of version in various files.
```

Comment:
```suggestion
This PR would contain an updated of version in various files.
```

Reply:
or maybe `This PR would update the older Kanister version reference in Kanister project.`.